guidance/models/llama_cpp/_llama_cpp.py

```diff
@@ -55,6 +57,30 @@ def __del__(self):
                 self.batch = None
                 llama_batch_free(batch)
 
+def detokenize(tokenizer: "LlamaTokenizer", tokens: list[int], special: bool = False, size: int = 32) -> bytes:
```

Comment:
I would say bump up the buffer to 256 by default; I don't know how they came up with 32